zaino-serve/src/rpc/service.rs

```diff
@@ -53,6 +55,37 @@ impl futures::Stream for RawTransactionStream {
     }
 }
 
+/// Stream of RawTransactions, output type of get_taddress_txids.
+pub struct CompactTransactionStream {
+    inner: ReceiverStream<Result<CompactTx, tonic::Status>>,
+}
+
+impl CompactTransactionStream {
+    /// Returns new instanse of RawTransactionStream.
```

Comment:
spelling mistake and also maybe type is wrong?
```suggestion
    /// Returns new instance of CompactTransactionStream.
```